GameServerLib/Config.cs

```diff
@@ -72,35 +72,17 @@ private void LoadConfig(Game game, string json)
             ContentPath = (string)gameInfo.SelectToken("CONTENT_PATH");
 
             // Read global damage text setting
-            IsDamageTextGlobal = (bool)gameInfo.SelectToken("IS_DAMAGE_TEXT_GLOBAL");
-
-            // Load items
-            game.ItemManager.AddItems(ItemContentCollection.LoadItemsFrom(
-                // todo: remove this hardcoded path with content pipeline refactor
-                $"{ContentPath}/LeagueSandbox-Default/Items"
-            ));
+            IsDamageTextGlobal = (bool) gameInfo.SelectToken("IS_DAMAGE_TEXT_GLOBAL");
 
             // Read the game configuration
             var gameToken = data.SelectToken("game");
             GameConfig = new GameConfig(gameToken);
 
-            // Read spawns info
-            ContentManager = ContentManager.LoadGameMode(game, GameConfig.GameMode, ContentPath);
-            var mapPath = ContentManager.GetMapDataPath(GameConfig.Map);
-            var mapData = JObject.Parse(File.ReadAllText(mapPath));
-            var spawns = mapData.SelectToken("spawns");
+            // Load data package
+            ContentManager = ContentManager.LoadDataPackage(game, GameConfig.DataPackage, ContentPath);
```

Comment:
May want to re-add some form of distinction between packages and modes (for another PR most likely).